pkg/reading/service.go

```diff
@@ -0,0 +1,62 @@
+package reading
+
+import (
+	"errors"
+	"fmt"
+	"github.com/TurnsCoffeeIntoScripts/jira-api-resource/pkg/configuration"
+	"github.com/TurnsCoffeeIntoScripts/jira-api-resource/pkg/helpers"
+	"github.com/TurnsCoffeeIntoScripts/jira-api-resource/pkg/http/rest"
+	"github.com/TurnsCoffeeIntoScripts/jira-api-resource/pkg/service"
+	"net/http"
+)
+
+type ServiceReadIssue struct {
+	issueId   string
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [struct field issueId should be issueID](https://app.codacy.com/app/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4570750)